<?xml version="1.0"?>
<launch>

	<node pkg="pcl_filters" type="statisticaloutlierremoval_filter" name="statisticaloutlierremoval_filter" output="screen" required="true"/>
 
</launch>
